### ISSUE TITLE
Feature (adjustment) request: Sort by time left

### ISSUE THREAD

**Issue by NiklasBr** [transmission/transmission#6648]
### What is the issue?

Current _sort by time left_ is a bit odd, it sorts like this from top down:

 1. 5 days left
 2. 7 hours left
 3. 25 minutes left
 4. Remaining time unknown
 5. Remaining time unknown
 6. Remaining time unknown
 7. ...

Or reversed from bottom up

 7. ...
 6. Remaining time unknown
 5. Remaining time unknown
 4. Remaining time unknown
 3. 25 minutes left
 2. 7 hours left
 1. 5 days left

It would be much more intuitive to treat _remaining time unknown_ as `infinity` rather than `0` (I suspect) so that from top down it would be:

 1. 25 minutes left
 2. 7 hours left
 3. 5 days left
 4. Remaining time unknown
 5. Remaining time unknown
 6. Remaining time unknown
 7. ...

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

4.0.5 (a6fe2a64aa)

**Comment by killemov** [comment]
I'm guessing that "Remaining time unknown" is a representation for null. Maybe do a quick scan here: https://github.com/transmission/transmission/tree/1855cdb7312acf903d37e0bb3a66501c2042fc44/qt ?

**Comment by NiklasBr** [comment]
Maybe this? It seems to indicate that it is NULL or -1 or some such value:

https://github.com/transmission/transmission/blob/1855cdb7312acf903d37e0bb3a66501c2042fc44/gtk/Torrent.cc#L518-L525

**Comment by killemov** [comment]
Nope, this is the confirmation of my assumption, not the comparator usually used for sorting.

**Comment by Coeur** [comment]
pull request welcomed

**Comment by killemov** [comment]
@Coeur Could it be that the sorting mechanism is actually part of the UI-element?
